tests/pytest_tests/unit_tests_old/tests_launch/test_kaniko_build.py

```diff
@@ -275,6 +275,4 @@ def test_build_image_success(
             "Created kaniko job wandb-launch-container-build-"
             in capsys.readouterr().err
         )
-        assert (
-            image_uri == "12345678.dkr.ecr.us-east-1.amazonaws.com/test-repo:b60e433c"
-        )
+        assert "12345678.dkr.ecr.us-east-1.amazonaws.com/test-repo" in image_uri
```

Comment:
Sort of unrelated to the rest of the change, but this test has been failing on my machine due to the hash being different and it's been driving me crazy for a while

Reply:
I think the hash might be different because of the python version

Reply:
I think it's because my username is different to the one used in CI, some debug prints showed that the user [here](https://github.com/wandb/wandb/blob/main/wandb/sdk/launch/builder/build.py#LL159C22-L159C22) is `timh`, which affects the output hash.